src/compiler/program.ts

```diff
@@ -203,18 +203,25 @@ namespace ts {
         return compilerHost;
     }
 
+    interface ComplierHostLikeForCache {

```

Comment:
typo:CompilerHostLikeForCache

Reply:
Also it would be nice to put all the CompilerHost-related types in one place and try to come up with meaningful names for them, or least relationships.

Reply:
good suggestion. Will add that to #28640